bin/dredd-hooks-php

```diff
@@ -23,11 +23,48 @@ if (!$loaded) {
     );
 }
 
+// Get options from the command line
+$options = getopt('', [
+    'host:',
+    'port:',
+    'force',
+]);
+
+// Extract parameters from options
+if (array_key_exists('host', $options)) {
+	$host = $options['host'];
+}
+if (array_key_exists('port', $options)) {
+	$port = $options['port'];
+}
+$force = array_key_exists('force', $options);
+
+// Remove parameters from the list of arguments
+foreach($options as $key => $value) {
+	if ($pos = array_search(sprintf('--%s=%s', $key, $value), $argv)) {
+		unset($argv[$pos]);
+		continue;
+	}
+	$posKey = array_search(sprintf('--%s', $key), $argv);
+	$posValue = array_search($value, $argv);
+	unset($argv[$posKey]);
+	if ( 1 == abs($posKey - $posValue)) {
+		unset($argv[$posValue]);
+		continue;
+	}
+}
```

Comment:
@ddelnano Yep. There is no easy way to do it. Or at least, I am not aware of one if it exists